sklearn/model_selection/_split.py

```diff
@@ -2175,29 +2127,19 @@ def train_test_split(*arrays, **options):
     if options:
         raise TypeError("Invalid parameters passed: %s" % str(options))
 
-    if test_size == 'default':
-        test_size = None
-        if train_size is not None:
-            warnings.warn("From version 0.21, test_size will always "
-                          "complement train_size unless both "
-                          "are specified.",
-                          FutureWarning)
-
-    if test_size is None and train_size is None:
-        test_size = 0.25
-
     arrays = indexable(*arrays)
 
+    n_samples = _num_samples(arrays[0])
+    n_train, n_test = _validate_shuffle_split(n_samples, test_size,
+                                              train_size,
```

Comment:
Nitpick: if I am not mistaken, ` train_size,` fits on the previous line under the 79 columns limit.

Reply:
how did you catch that :D ?

Reply:
I think I have a built-in pep8 filter in my visual cortex.